setup.cfg

```
@@ -0,0 +1,39 @@
+[metadata]
+name = icubam
+description = "ICU Bed Activity Monitor."
+version = 0.1.0
```

```suggestion
version = 0.1.2
```
(same as in resources/config.toml)